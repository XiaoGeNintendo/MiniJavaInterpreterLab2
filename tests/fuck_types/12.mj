int main(){
    assert(null!=3); //Fails
    return 0;
}