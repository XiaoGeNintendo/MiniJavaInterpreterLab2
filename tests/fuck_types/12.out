Process exits with the code 34.